hardhat.config.ts

```diff
@@ -0,0 +1,35 @@
+import { HardhatUserConfig } from "hardhat/config";
+import "@nomicfoundation/hardhat-toolbox";
+import "@nomiclabs/hardhat-ethers";
+
+// The next line is part of the sample project, you don't need it in your
+// project. It imports a Hardhat task definition, that can be used for
+// testing the frontend.
+import "./tasks/accounts.ts";
+
+const config: HardhatUserConfig = {
+  solidity: "0.8.17",
+  networks: {
+    hardhat: {
+      chainId: 31337, // We set 1337 to make interacting with MetaMask simpler
+      accounts: {
+        mnemonic: "test test test test test test test test test test test junk",
```

Comment:
Let's change the mnemonic to a different one from gs.

Reply:
Done

Reply:
Should I use a specific mnemonic or whatever we want?

Reply:
use "cat cat cat ... cat cat"